src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -93,20 +95,23 @@ internal TypeSymbolWithAnnotations ToType()
         /// </summary>
         public readonly bool IsAnnotated;
 
+        public readonly bool TreatUnconstrainedTypeParameterAsNullable;
```

Comment:
Do we really need to expose this publicly?

Reply:
Changed. All uses are private to this type (or nested types in the case of `Builder` and `Extensions).

---
In reply to: [208372320](https://github.com/dotnet/roslyn/pull/29127#discussion_r208372320) [](ancestors = 208372320,208360056)